code/modules/mapping/map_template.dm

```diff
@@ -84,6 +84,13 @@
 	if(T.y+height > world.maxy)
 		return
 
+	var/list/border = block(locate(max(T.x-1, 1),			max(T.y-1, 1),			 T.z),
+							locate(min(T.x+width+1, world.maxx),	min(T.y+height+1, world.maxy), T.z))
+	for(var/L in border)
+		var/turf/turf_to_disable = L
+		SSair.remove_from_active(turf_to_disable) //stop processing turfs along the border to prevent runtimes, we return it in initTemplateBounds()
```

Comment:
how does it prevent runtimes?

Reply:
Border turfs dont know that we change space to wall, because we halt it before all loaded, and plating dont create air in self before all initialized.
It try to interract with uninitialized turfs and this create runtimes like:
`[2019-07-10 06:25:47.949] runtime error: Cannot execute null.archive().`
and
`[06:25:49] Runtime in gas_mixture.dm,380: Cannot read null.gases`

Here i halt processing of outer border turfs, it return in work when all template initializes.

Reply:
Say me plese, this one work?